tests/dflydev/tests/util/antPathMatcher/AntPathMatcherTest.php

```diff
@@ -95,7 +100,12 @@ public function testMatchesProvider()
                 'com/foo/',
                 array('com/foo/bar.jsp','com/foo/bar/baz.jsp',),
                 array('com.txt', 'com/foo.txt'),
-            )
+            ),
+            array(
+                'com/**',
+                array('com/foo'),
+                array('com2/foo', '_com/foo'),
```

Comment:
While responding above, I realized it'd be helpful to add a second test case since we're anchoring the regex on both sides. Testing that `com/**` doesn't match `com2/foo` only concerns the `$` anchor. The second `_com/foo` test case will exercise the `^` anchor.